core/iwasm/compilation/aot_compiler.c

```diff
@@ -2743,7 +2760,7 @@ aot_compile_func(AOTCompContext *comp_ctx, uint32 func_index)
                     case SIMD_v128_store:
                     {
                         read_leb_uint32(frame_ip, frame_ip_end, align);
-                        read_leb_uint32(frame_ip, frame_ip_end, offset);
+                        read_leb_mem_offset(frame_ip, frame_ip_end, offset);
```

Comment:
Should also replace the `read_leb_uint32(frame_ip, frame_ip_end, offset);` for L3009, L3023, L3035. 

And I think it's best to leave a comment before `case WASM_OP_SIMD_PREFIX` and `case WASM_OP_ATOMIC_PREFIX` with: `TODO: in the future, replace the offset type in aot_compile_op_xxx for threads and SIMD opcodes compilation`

Reply:
hmm, I think I'd rather just change this now instead of leaving todos